promptsource/templates/GEM/totto/templates.yaml

```diff
@@ -0,0 +1,166 @@
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %}
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Guess the URL from which this table has been extracted!
+
+      |||
+
+      {{table_webpage_url}}
+
+      {% endif %}
+
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table webpage url
+    reference: ''
+  7a2fb34a-c792-41b3-9682-fcda15f1d100: !Template
+    answer_choices: null
+    id: 7a2fb34a-c792-41b3-9682-fcda15f1d100
+    jinja: '{% if table_section_title != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      {% if table_section_text != "" %}
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %},
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you guess the table''s section title?
+
+      |||
+
+      {{table_section_title}}
+
+      {% endif %}
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table section title
+    reference: ''
+  a78e0a87-c722-463f-bef6-c231108f8fdf: !Template
+    answer_choices: null
+    id: a78e0a87-c722-463f-bef6-c231108f8fdf
+    jinja: '{% if table_section_text != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      Table Section Title: {{table_section_title}},
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you guess the table''s section text?
+
+      |||
+
+      {{table_section_text}}
+
+      {% endif %}
+
+      '
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - BLEU
+      - ROUGE
+      original_task: false
+    name: guess the table section text
+    reference: ''
+  d2fd58d2-b7dc-4eae-b1a4-63b89126445c: !Template
+    answer_choices: null
+    id: d2fd58d2-b7dc-4eae-b1a4-63b89126445c
+    jinja: '{% if sentence_annotations[0]["final_sentence"] != "" %}
+
+      Given the following details:
+
+      Table Page Title: {{table_page_title}},
+
+      Table Section Title: {{table_section_title}},
+
+      {% if table_section_text != "" %}
+
+      Table Section Text: {{table_section_text}},
+
+      {% endif %}
+
+      and the following original text describing the highlighted cell of the table:
+
+      {{sentence_annotations[0]["original_sentence"]}},
+
+      Can you generate a final text taking into account the context from the details
+      given above?
+
+      |||
+
+      {{sentence_annotations[0]["final_sentence"]}}
```

Comment:
This prompt reflects the original task best but again I think requires formatting more like the original paper